### PR TITLE
Using kind:jobs for uperf/fio

### PR DESCRIPTION
**Pull request by aakarshg**
Also updated tests to look for job completion.
uperf: clients and fio both don't need to be running forever, so switched to jobs

### CODE REVIEW

**Comment by aakarshg** [comment]
rerun smoke

File: roles/uperf-bench/tasks/main.yml [hunk 1]
```diff
@@ -81,30 +81,33 @@
...
-        kind: Pod
-        apiVersion: v1
+        kind: Job
+        apiVersion: batch/v1
         metadata:
           name: '{{ meta.name }}-uperf-client-{{item.status.podIP}}'
           namespace: '{{ meta.namespace }}'
-          labels:
-            app : uperf-bench-client
         spec:
```
Comment (jtaleric) [inline roles/uperf-bench/tasks/main.yml]: Should we use : `ttlSecondsAfterFinished` to automatically cleanup?

**Comment by aakarshg** [comment]
rerun smoke

File: roles/uperf-bench/tasks/main.yml (hunk 1 above)
Comment (aakarshg) [reply roles/uperf-bench/tasks/main.yml]: Yes I think that'll be helpful, also I was thinking we can clean up server as soon as the client job completes successfully. Thus we'll keep the client pod but delete the serve pod once client finishes running. what'd you think?

Comment (jtaleric) [reply roles/uperf-bench/tasks/main.yml]: Hm - I would think the other way around. Client completes, deletes. Server deletes? Not sure if the order really matters...

Comment (aakarshg) [reply roles/uperf-bench/tasks/main.yml]: After discussion with @jtaleric on bluejeans, we've agreed upon deleting the server once the client job completes and use the `ttlSecondsAfterFinished` for client job.

**Comment by aakarshg** [comment]
rerun smoke

**Comment by aakarshg** [comment]
rerun smoke kube

**Comment by aakarshg** [comment]
rerun smoke kube

**Comment by aakarshg** [comment]
rerun smoke kube

**Comment by aakarshg** [comment]
rerun smoke kube

**Comment by aakarshg** [comment]
rerun smoke kube

**Comment by aakarshg** [comment]
rerun smoke kube

**Comment by aakarshg** [comment]
rerun smoke kube

**Comment by aakarshg** [comment]
rerun smoke kube

**Comment by aakarshg** [comment]
rerun smoke kube

File: roles/fio-bench/tasks/main.yml [hunk 2]
```diff
@@ -14,28 +14,30 @@
...
+        template:
+          metadata:
+            labels:
+              app: fio-bench
+          spec:
+            containers:
+            - name: bench-server
+              image: "quay.io/jtaleric/fio:testing"
+              command: ["/bin/sh","-c"]
+              args: ["cat /tmp/fio/fiojob;fio /tmp/fio/fiojob"]
```
Comment (jtaleric) [inline roles/fio-bench/tasks/main.yml]: I don't think this reads as well, can you change it back so it reads easier?

**Comment by aakarshg** [comment]
rerun smoke

**Pull request merged**